addon/src/lib/middleware/Hub.js

```diff
@@ -14,8 +14,8 @@ export default class Hub {
   dispatch: Dispatch;
   ports: Set<EventEmitter>;
   constructor() {
-    this.dispatch = () =>
-      console.error('Hub cannot use dispatch() before middleware()');
+    this.dispatch = () => // eslint-disable-next-line no-console
+    console.error('Hub cannot use dispatch() before middleware()');
```

Comment:
This falls into the "should never happen" bin. I'll even tempt fate and say it'd never make it past QA. So its just a hint to me if I grossly bork it up.